Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift

```diff
@@ -186,6 +184,14 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
         }
     }
 
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        if case .balances(items: let items) = sections[indexPath.section] {
```

Comment:
seem to conflict this pr with https://github.com/gnosis/safe-ios/pull/1844

Reply:
true. It was not merged in the release branch so I didn't see it

Reply:
yes, please revert 😅